### PR TITLE
Add pretend recipe.

### PR DESCRIPTION
**Pull request by sodre**
Closes conda-forge/staged-recipes#1331

### CODE REVIEW

File: recipes/pretend/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - pretend
+
+about:
+  home: https://github.com/alex/pretend
+  license: BSD
```
Comment (jakirkham) [inline recipes/pretend/meta.yaml]: Please add number of clauses. Believe this should be `BSD 3-Clause`.

File: recipes/pretend/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  license_file: LICENSE.rst
+  summary: 'A library for stubbing in Python'
+  license_family: BSD
```
Comment (jakirkham) [inline recipes/pretend/meta.yaml]: Can we move this up with the license metadata please?

**Comment by jakirkham** [comment]
Thanks @sodre. Couple simple things and then I think it is good to go.

**Comment by sodre** [comment]
No problem, both issues are fixed.

**Comment by jakirkham** [comment]
Thanks @sodre.

**Pull request merged**